src/mito2/src/memtable/merge_tree/partition.rs

```diff
@@ -578,12 +593,16 @@ impl Inner {
         }
     }
 
-    fn find_key_in_shards(&self, primary_key: &[u8]) -> Option<PkId> {
+    fn find_key_in_shards(&self, short_key: &[u8]) -> Option<PkId> {
```

Comment:
```suggestion
    fn find_key_in_shards(&self, sparse_key: &[u8]) -> Option<PkId> {
```

Reply:
> fn find_key_in_shards(&self, short_key: &[u8]

I think better to rename it back to `primary_key`